backend/src/main/java/com/htecgroup/skynest/controller/LambdaController.java

```diff
@@ -67,6 +67,51 @@ public List<LambdaType> getAllLambdas() {
     return Arrays.stream(LambdaType.values()).collect(Collectors.toList());
   }
 
+  @Operation(summary = "Deactivate lambda for bucket")
+  @ApiResponses(
+      value = {
+        @ApiResponse(
+            responseCode = "204",
+            description = "Lambda successfully deactivated for given bucket"),
+        @ApiResponse(
+            responseCode = "404",
+            description = "Bucket not found",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"Bucket not found\"],"
+                                + " \"status\": \"401\","
+                                + " \"timestamp\": \"2022-06-07 16:18:12\"}")
+                  })
+            }),
+        @ApiResponse(
+            responseCode = "401",
+            description = "Unauthorized request",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"Access denied\"],"
+                                + " \"status\": \"401\","
+                                + " \"timestamp\": \"2022-06-07 16:18:12\"}")
+                  })
+            }),
+      })
+  @PutMapping("/bucket/{bucketId}/deactivate")
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  public void deactivateLambdaForBucket(
+      @PathVariable UUID bucketId, @RequestParam LambdaType lambda) {
+    bucketService.deactivateLambda(bucketId, lambda);
+    log.info("Deactivated lambda {} for bucket {}", lambda.toString(), bucketId.toString());
```

Comment:
Maybe to log also which owner deactivated labmda..

Reply:
That would be nice, log the user that activated/deactivated the lambda. Sure